etsin_finder/frontend/js/stores/view/qvain.js

```diff
@@ -708,6 +738,14 @@ class Qvain {
       })
     }
 
+    // Languages of dataset
+    this.datasetLanguage = []
```

Comment:
I'm getting a prop type warning because `datasetLanguage` is an array. I think this part is supposed to be:
```suggestion
    this.datasetLanguage = undefined
    this.datasetLanguageArray = []
```

Reply:
Thanks, totally should be initially undefined instead of empty array!